pages/dkp/kaptain/1.3.0/install/air-gapped-dkp/index.md

```diff
@@ -51,6 +51,7 @@ Use the following instructions to install Spark Operator from Kommander Catalog
 ### Load the Docker images into your Docker registry
 
 1. Download the image bundle file:
+<!-- NEW TAR because of Minio -->
```

Comment:
This is where we need them to extract the NOTICES.txt and the image bundle which will also be a tar.
```suggestion
  Extract the image bundle archive and other files before use.
  
  ```bash
  tar -xvf kaptain-air-gapped-1.3.0.tar.gz
  ```
    <!-- NEW TAR because of Minio in this whole code block?-->
```